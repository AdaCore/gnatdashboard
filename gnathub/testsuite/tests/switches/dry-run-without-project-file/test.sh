gnathub --dry-run | grep "sonar-config"
