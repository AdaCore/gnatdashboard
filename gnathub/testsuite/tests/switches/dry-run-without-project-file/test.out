plugin-runner: sonar-config plug-in is available